### ISSUE TITLE
cross compile support

### ISSUE THREAD

**Issue by blaggacao** [nrdxp/nixflk#86]
**Is your feature request related to a problem? Please describe.**
On my attempt to cross-compile for a RaspberryPi host, I [got frustrated](https://github.com/nrdxp/nixflk/pull/73#issuecomment-758299125) with a refusal to cross compile.

(I'm a layman, more than others who claim it, but) I believe, the reason is that nixflk currently doesn't hanle the `crossSytem` attribute, which I reference from my own context and ongoing line of thought [here](https://github.com/numtide/flake-utils/pull/18/files#diff-79c5531512c8fa0e4522e5d82d96fc2d93b75040e0a197cb0a94c7cb0b2c19b3R81-R82) and [here](https://github.com/numtide/flake-utils/pull/18/files#diff-79c5531512c8fa0e4522e5d82d96fc2d93b75040e0a197cb0a94c7cb0b2c19b3R109).

**Describe the solution you'd like**
Not sure how to sort this out short vs long term. A consolidated numtide/flake-utils backed solution would be ideal. (see the linked passages).

**Describe alternatives you've considered**
Too little a knowledge to be serious on alternatives.

**Additional context**

I'm trying to get some [PI-based "home-automation" going](https://github.com/nrdxp/nixflk/pull/73).

**Comment by nrdxp** [comment]
Is this a duplicate of #72? Cross compiling packages already works by pulling from `pkgsCross` as usual, so at the very least, the title may need revision.

I am also able to build for aarch by adding:
```nix
{
  boot.binfmt.emulatedSystems = [ "aarch64-linux" ];
}
```

So I'm not sure if this may solve the issue, at least for building aarch packages on x86.

**Comment by blaggacao** [comment]
> Is this a duplicate of #72?

Indeed! Or at least subsumable thereunder. Thanks for pointing this out.
